poc/vdaf_mastic.py

```diff
@@ -0,0 +1,370 @@
+import sys
+sys.path.append('draft-irtf-cfrg-vdaf/poc')
+
+from common import Unsigned, front, vec_add, vec_sub
+from flp_generic import FlpGeneric
+import hashlib
+from typing import Optional, Union
+from vdaf import Vdaf, test_vdaf
+from vdaf_poplar1 import Poplar1
+from vidpf import Vidpf
+from xof import XofShake128
+
+
+USAGE_QUERY_RAND = 0
```

Comment:
Nit: Can you add a comment to describe what these three are? Also, maybe an `enum` would be cleaner?